rigs/heft-web-rig/shared/webpack-base.config.js

```diff
@@ -218,7 +218,7 @@ function createWebpackConfig({ env, argv, projectRoot, configOverride, extractCs
       port: 8080
     },
     // See here for documentation: https://webpack.js.org/configuration/devtool
-    devtool: production ? undefined : 'eval-source-map',
+    devtool: production ? undefined : 'source-map',
```

Comment:
This should be proposed in a different PR. See https://github.com/microsoft/rushstack/issues/4076

Reply:
removed